src/api/comments.ts

```diff
@@ -0,0 +1,23 @@
+export const getPostComments = (postId: number) => {
+  return fetch(`https://mate.academy/students-api/comments?postId=${postId}`)
+    .then(response => response.json());
+};
+
+export const deletePostComments = (commentId: number) => {
```

Comment:
```suggestion
export const deletePostComment = (commentId: number) => {
```
Ты же один комментарий удаляешь